definition-src/Game/LambdaHack/Definition/Defs.hs

```diff
@@ -14,7 +14,6 @@ module Game.LambdaHack.Definition.Defs
   ) where
 
 import Prelude ()
-
```

Comment:
please keep the import blocks separated

Reply:
Did you miss these, because they required "Load more"?

Reply:
this was merge error on my end. will fix